fjord/feedback/views.py

```diff
@@ -44,23 +46,39 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    if waffle.flag_is_active(request, 'thankyou'):
-        template = 'feedback/thanks.html'
-        try:
-            opinion_id = request.session.get('opinion_id')
-            user_opinion = Response.objects.get(id=opinion_id)
-        except Response.ObjectDoesNotExist:
-            pass
-        else:
-            word_count = len(user_opinion.description.split())
-            if (user_opinion.locale == 'en-US' and
-                    not user_opinion.happy and
-                    word_count >= 7):
-                template = 'feedback/thanks_sad.html'
```

Comment:
This code regarding sentiment, locale and word count check is getting moved to the SUMO Search Suggest API suggest provider.